frontend/src/utils/helpers.js

```diff
@@ -95,3 +95,46 @@ export const getFiscalYear = () => {
 };
 
 export const formatParamStringToArray = (param) => (param ? param.split(",").filter((x) => x) : []);
+
+// This method sorts codes of the for '#.#.# - Lorem Ipsum'
+// where the number of integers is variable and the text is optional
+// TODO: In order to sort incidents this will need to be modified to deal with
+// parentheses
+export const compareCodes = (a, b) => {
+  // Returns the first match that is non-null.
+  const regexParse = (input) =>
+    input.match(/(?<first>[0-9]+)\.(?<second>[0-9]+)\.(?<third>[0-9]+)/) ||
+    input.match(/(?<first>[0-9]+)\.(?<second>[0-9]+)/) ||
+    input.match(/(?<first>[0-9]+)/);
+  const aCodes = regexParse(a);
+  const bCodes = regexParse(b);
+  if (!aCodes) return -1;
+  if (!bCodes) return 1;
+  let aInt;
+  let bInt;
+  const k = Math.min(aCodes.length, bCodes.length);
+  // Compares then non-null parts of two strings of potentially different lengths e.g 1.11 and 1.4.12
+  for (let i = 1; i < k; i += 1) {
+    aInt = parseInt(aCodes[i], 10);
+    bInt = parseInt(bCodes[i], 10);
```

Comment:
Reason for using `parseInt` instead of `Number`?

Reply:
It was the one I found, is there a reason we want Number?